examples/templates/docker/build/Dockerfile

```diff
@@ -1,3 +1,17 @@
 FROM ubuntu
 
-RUN apt-get update && apt-get install -y curl wget git vim golang
+RUN apt-get update \
+	&& apt-get install -y \
+	curl \
+	git \
+	golang \
+	sudo \
+	vim \
+	wget
```

Comment:
nit: WDYT about cleaning the directory `/var/lib/apt/lists` to reduce the image size?

Reply:
Definitely a good suggestion, normally I'd do it but I was thinking perhaps it's nice if you can immediately do a `sudo apt install thing` without `apt update` first. I can go ahead with cleaning it up though.

Reply:
Alternatively, you can leave it commented, so somebody can quickly enable the command.